python/src/etos_api/__init__.py

```diff
@@ -56,3 +57,5 @@
     FastAPIInstrumentor().instrument_app(
         APP, tracer_provider=PROVIDER, excluded_urls="selftest/.*,logs/.*"
     )
+
+REG = RegisterProviders()
```

Comment:
Why is this added here?

Reply:
The initial idea was to place it in main.py. However this file seems to be the actual entry point that imports from main.py. 

Should this be moved to main.py or some other place?